packages/mjml-button/src/index.js

```diff
@@ -51,85 +92,116 @@ class Button extends Component {
   styles = this.getStyles()
 
   getStyles () {
-    const { mjAttribute, defaultUnit } = this.props
+    const {mjAttribute} = this.props,
+      width = mjAttribute('width'),
```

Comment:
Hmm ye maybe i'll stop doing that for my own projects as well :D

You can start clicking from the outside of the button to select what's inside right?
Anyway it wont take too much time to code so.. your call :)